flake.nix

```diff
@@ -0,0 +1,32 @@
+{
+  description = "AWS-LC is a general-purpose cryptographic library";
+
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-24.11";
```

Comment:
Will we have to keep this up to date? Is there an option for latest?

Reply:
Latest is essentially a development branch and comes with the risk of breakage. There are only 2 releases a year (similar to Canonical's cadence), so the version bumps can be infrequent.